### PR TITLE
S3 validate

### PR DESCRIPTION
**Pull request by ddhar28**
- Works for s3 url of the form : s3://bucketname/key
- Files are selected based on a prefix, e.g. story, section, etc. as selected by user
- s3 access key and secret will be picked from config.js file

### CODE REVIEW

File: client/src/migrator/validation-form.js [hunk 1]
```diff
@@ -156,6 +173,14 @@ function InputField({validateType, onInput, userData}) {
               size={3000000}
```
Comment (Robert-Christopher) [inline client/src/migrator/validation-form.js]: can we make this a config ?

File: server/src/config.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const AWSCredentials = {
```
Comment (Robert-Christopher) [inline server/src/config.js]: This should not be hard coded

File: server/src/lib/handlers/intermediate-file-validator.ts [hunk 3]
```diff
@@ -1,31 +1,24 @@
...
-    'editor-types.d.ts'
-  );
+import S3 from 'aws-sdk/clients/s3'
+import {validator, asyncReadStream, typesPath} from '../utils/validator'
+import { AWSCredentials } from '../../config'
 
 export function textInputValidator(req: Request, res: Response): void {
   const { type, data } = req.body;
-  let result: Object = {}
+  let result: any = {}
```
Comment (Robert-Christopher) [inline server/src/lib/handlers/intermediate-file-validator.ts]: Why is this `any` ?

Comment (ddhar28) [reply server/src/lib/handlers/intermediate-file-validator.ts]: @Robert-Christopher changed the types. Please check

**Pull request merged**